libr/core/cmd_print.c

```diff
@@ -1613,37 +1625,42 @@ static void cmd_print_format(RCore *core, const char *_input, const ut8* block,
 				r_print_format (core->print, core->offset,
 					core->block, core->blocksize, name, mode, NULL, NULL);
 			}
+stage_name:
 			free (name);
 		}
 	} else {
-		/* This make sure the structure will be printed entirely */
+		/* Make sure the structure will be printed entirely */
 		const char *fmt = r_str_trim_head_ro (input + 1);
 		int struct_sz = r_print_format_struct_size (core->print, fmt, mode, 0);
 		int size = R_MAX (core->blocksize, struct_sz);
 		ut8 *buf = calloc (1, size);
 		if (!buf) {
-			eprintf ("cannot allocate %d byte(s)\n", size);
+			eprintf ("Cannot allocate %d byte(s) for an entire structure\n", size);
 			goto stage_left;
 		}
 		memcpy (buf, core->block, core->blocksize);
-		/* check if fmt is '\d+ \d+<...>', common mistake due to usage string*/
-		bool syntax_ok = true;
+
 		char *args = strdup (fmt);
 		if (!args) {
-			r_cons_printf ("Error: Mem Allocation.");
-			free (args);
-			goto stage_left;
+			eprintf ("Cannot allocate (%d) byte(s) for a format string\n", (int)strlen(fmt));
```

Comment:
space before (

Reply:
btw, there are different styles for spaces after type conversion like `(int)`, both are widely used. Even mixed on the same line:

https://github.com/radareorg/radare2/blob/5.0.0/libr/core/cmd_print.c#L5243

```
} else if (core->blocksize_max < use_blocksize && (int) use_blocksize > -(int)core->blocksize_max) {
```